RESTserver/src/main/java/com/softserve/if072/restservice/controller/StorageController.java

```diff
@@ -64,38 +64,23 @@ public void delete(@PathVariable int id, HttpServletResponse response) {
         }
     }
 
-    @GetMapping(value = "/{id}")
-    @ResponseBody
-    @ResponseStatus(value = HttpStatus.OK)
-    public Storage getById(@PathVariable int id, HttpServletResponse response) {
-        try {
-            Storage store = storageService.getById(id);
-            LOGGER.info(String.format("Storage with id %d was retrieved", id));
-            return store;
-        } catch (DataNotFoundException e) {
-            response.setStatus(HttpServletResponse.SC_NOT_FOUND);
-            LOGGER.error(String.format("Storage with id %d was not found", id), e);
-            return null;
-        }
-    }
 
     @PostMapping(value = "/")
-    @ResponseStatus(value = HttpStatus.OK)
-    public void insert(@RequestBody Storage storage){
+    @ResponseStatus(value = HttpStatus.CREATED)
+    public void insert(@RequestBody Storage storage) {
         storageService.insert(storage);
         LOGGER.info("New Storage was created");
     }
 
     @PutMapping(value = "/")
     @ResponseStatus(value = HttpStatus.OK)
     public void update(@RequestBody Storage storage, HttpServletResponse response) {
-        int id = storage.getId();
         try {
             storageService.update(storage);
-            LOGGER.info(String.format("Store with id %d was updated", id));
+            LOGGER.info("Store was updated");
         } catch (DataNotFoundException e) {
             response.setStatus(HttpServletResponse.SC_NOT_FOUND);
-            LOGGER.error(String.format("Cannot update Storage with id %d", id), e);
```

Comment:
why did you remove useful information from the logs? 
This is related to this row and above rows